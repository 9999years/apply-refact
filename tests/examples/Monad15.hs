yes = do if a then forM x y else sequence z q; return ()