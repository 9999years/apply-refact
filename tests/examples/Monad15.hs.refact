[("tests/examples/Monad15.hs:1:10: Error: Use Use simple functions\nFound:\n  if a then forM x y else sequence z q\nWhy not:\n  if a then forM_ x y else sequence_ z q\n",[Replace {rtype = Expr, pos = SrcSpan {startLine = 1, startCol = 20, endLine = 1, endCol = 24}, subts = [], orig = "forM_"},Replace {rtype = Expr, pos = SrcSpan {startLine = 1, startCol = 34, endLine = 1, endCol = 42}, subts = [], orig = "sequence_"}])]
