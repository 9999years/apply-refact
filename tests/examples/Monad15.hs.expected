yes = do if a then forM_ x y else sequence_ z q; return ()
